### DISCUSSION TITLE
how to handle Page loading spinner in webdriverio

### DISCUSSION THREAD

**Discussion by prasadqa** [webdriverio/webdriverio#8171] [Community]
how to handle Page loading spinner in webdriverio ? 
i have tried below solution but it is working fine when execute in local (same system) but it is failing when execute in sauce labs 
var elem = await $('//ngx-spinner/div');
        console.log("DDDDD=====>", elem.isDisplayed());
        await browser.waitUntil(
            async () => !elem.isDisplayed(),
            {
                timeout: 600000,
                timeoutMsg: 'Wait T0 Disappear - Time out '
            }
        );

**Comment by christian-bromann** [comment]
Try:
```ts
await $('//ngx-spinner/div').waitForDisplayed({ reverse: true })
```
